dispatch-service/src/test/resources/application-test.yml

```diff
@@ -13,6 +13,15 @@ swim:
       destination-binding: dms-out
       mail-addresses:
         - test-meta@example.com
+    - name: test-meta-dest
+      bucket: test-bucket-3
+      path: test3
+      recursive: true
+      requires-metadata: true
+      destination-via-metadata: true
+      destination-binding: dms-out
```

Comment:
it just looks confusing to have `destination-via-metadata: true` and then `destination-binding: dms-out` right underneath. It is not obvious what has precedence. 
Maybe we should rename `destination-binding` to `default-destination-binding`  - what do you think?

Reply:
We discussed that in https://github.com/it-at-m/swim/issues/27#issuecomment-2612508867 and in my opinion `default-destination-binding` suggests, that there is always another resolution, which is not true but only if `destination-via-metadata: true` is set.
I understand your point. I don't really like either.

Reply:
README would definitely help, but still does not make it more readable in the yml. If we leave it this way, we should at least add in-line-comments directly in the yml (similar as you've already done in `application.yml`). Alternatively we could also support both `destionation-binding` and `default-destination-binding`, mapping it to the same internal field or similar.

Reply:
Alternative would be to change `destinationViaMetadata` to `overwriteDestinationViaMetadata`.

> Alternatively we could also support both destionation-binding and default-destination-binding, mapping it to the same internal field or similar.

I don't think that can be done as the complete properties parsing is done by Spring and this might just make the autocompletion and parsing more difficult. (If at all possible, which I doubt)

Reply:
What I mean is something like this:

![image](https://github.com/user-attachments/assets/7425a816-d324-44bc-bd9d-6a9380dc707c)

meaning that both exist interchangeably as option.

Reply:
OK, let's go with `overwriteDestinationViaMetadata` and please include some comment in the README